config/flags_service_openvpn.go

```diff
@@ -50,13 +50,16 @@ var (
 	FlagOpenVPNPriceMinute = cli.Float64Flag{
 		Name:  "openvpn.price-minute",
 		Usage: "Sets the price of the OpenVPN service per minute.",
-		Value: 0.0001,
```

Comment:
This is how it looks now in the help message:
![image](https://user-images.githubusercontent.com/8612618/81151178-8cec6d00-8fa2-11ea-8490-64b5b85d0084.png)

It says the default price for OpenVPN service is 0.